code-challenges.js

```diff
@@ -7,43 +7,68 @@
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
 
 // Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+
+// console.log(temp1())
+// Create a function called boilingPoint.
+// Input different temps.
+// use conditional statement to make an evaluation on the size.
+
+
+
+
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
-const myNumbers1 = [3, 7, 0, 36, -9]
-const myNumbers2 = [8, -7, 42, 9, 13]
+// const myNumbers1 = [3, 7, 0, 36, -9]
+// const myNumbers2 = [8, -7, 42, 9, 13]
+//
+// console.log(myNumbers1.concat(myNumbers2).length)
 
+//Use an accessor to combine two arrays.
+//Use length to return the length.
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
 const currentCohort = "Bravo 2022"
 
+console.log(currentCohort.reverse())
 
-// --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
-
-const myArray = [13, 34, 5, 10, 27, 42]
+//--------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
+// const myArray = [13, 34, 5, 10, 27, 42]
+//
+// for(let i = 0; i < myArray.length; i ++) {
+//   if (myArray [i] % 2 == 0) {
+//     console.log(`${myArray[i]} is an even number`)
+//   } else { console.log(`${myArray[i]} is an odd number`)
+//   }
+// }
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
-// Set one:
-const number1 = 58
-const number2 = 100
-
-// Set two:
-const number3 = 27
-const number4 = 24
+//  Set one:
+// const number1 = 58
+// const number2 = 100
+//
+//  Set two:
+// const number3 = 27
+// const number4 = 24
+//
+// const newNumber = () => {
+//   return (number2 - number1)
+// }
+// console.log(newNumber())
```

Comment:
I don't see any pseudocode for this problem either.  How can this function become more dynamic?  Instead of hardcoding the variables number2 and number1, can you pass any 2 numbers into this function, find which one is greater and then subtract the smaller one from the larger one?